lib/features/nems_core/nems_core.module

```diff
@@ -30,9 +30,12 @@ const NEMS_CORE_BLOCK_ACCESS_PERMISSIONS = array(
  * Implements hook_nexteuropa_token_token_handlers().
  */
 function nems_core_nexteuropa_token_token_handlers() {
-  return array(
-    'handler_name' => '\Drupal\nems_core\FileUrlTokenHandler',
-  );
+  // This is shared functionality with mbp_sync, so we check for conflict.
+  if (!class_exists('Drupal\mbp_sync\FileUrlTokenHandler')) {
```

Comment:
I do not agree with this approach. The correct way is to prefix the classname with the module name, so this would become NemsCoreFileUrlTokenHandler.